docs/solutions/k8s-on-dcos.md

```diff
@@ -0,0 +1,271 @@
+# Blueprint - Kubernetes on the Mesosphere DCOS
+
```

Comment:
Agreed and indeed, that was one of the options, as discussed with Craig. We settled on this version.